src/App.tsx

```diff
@@ -47,9 +51,152 @@ function App() {
 
     }, [selectedFile]);
 
+
+    function isLastDay(date: string) {
+        let d = new Date(date)
+        let test = new Date(d.getTime())
+        let month = test.getMonth();
+        test.setDate(test.getDate() + 1);
+        return test.getMonth() !== month;
+    }
+
+
+    const sortedData = useMemo(() => {
+
+        if (sortBy === 'daily') {
+            return data
+        } else {
```

Comment:
Here you can go with an early return. Means, you can and should remove the else here. 
https://levelup.gitconnected.com/how-you-can-avoid-using-else-in-your-code-871197a1adbc